Gemfile.lock

```diff
@@ -73,6 +79,7 @@ DEPENDENCIES
   jekyll (~> 4.1.0)
   jekyll-feed (~> 0.12)
   minima (~> 2.5)
+  thin
```

Comment:
I see that this was added to the gemfile.lock but is it not present on the Gemfile.

Should it replace `webrick`?

Nevertheless these changes should not be bundled together with this PR for the youtube player.

Reply:
@tiagovarela  I got an error running bundle exec jekyll serve , while getting this up on local and it asked for me to install thin. How do I separate the changes on gitignore from this PR in that case.

Reply:
If you are getting an error we should revisit the local development setup. 

Since github does not use the web server from the Gemfile to host the github page it should have no impact on the feature being deployed here.

So while you can change the file to work locally they should not be committed in this PR and after we review the local development setup we can create another PR with just those changes.

Reply:
On it.